operator/charts/netchecks/artifacthub-repo.yml

```diff
@@ -0,0 +1,5 @@
+# Artifact Hub repository metadata file
+repositoryID: The ID of the Artifact Hub repository where the packages will be published to (optional, but it enables verified publisher)
```

Comment:
@hardbyte  an ID is needed here—I believe it can be grabbed from https://artifacthub.io/control-panel/repositories